### ISSUE TITLE
Invalid type expression form (list literal) not rejected when used in Annotated

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#6863]
The following should be flagged as an error, but it's not.

```python
Bad1: Annotated[[int, str], ""]  # Type error: invalid type expression
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.345, which I just published. It will also be included in a future release of pylance.
